third_party/sl4a/src/main/java/com/google/android/mobly/snippet/facade/FacadeManager.java

```diff
@@ -30,35 +30,24 @@
 
 public class FacadeManager extends SnippetManager {
 
-  private final Context mContext;
   private int mSdkLevel;
 
-  public FacadeManager(int sdkLevel, Context context,
-                       Collection<Class<? extends Snippet>> classList) {
-    super(context, classList);
+  public FacadeManager(int sdkLevel, Collection<Class<? extends Snippet>> classList) {
```

Comment:
I find Context to be useful, especially when writing facades.
Should we keep it or is there a better way to get it other than passing via constructor.
Also, should this class be renamed since we don't have the concept of 'Facade' in snippet?